pages/OAuth2Server/OAuth2ServerExample.md

```diff
@@ -0,0 +1,51 @@
+---
+title: OAuth2Server Usage Example
+permalink: /oauth2server/oauth2server_example
+---
+
+# Example of usage of the `OAuth2Server`
+
+Find [this repository](https://github.com/pedroetb/node-oauth2-server-example) that explains in detail how to use the `OAuth2Server`. However, find below some examples as a fast guide.
+
+First of all, all authentication requests must include the `Authorization` HTTP header: 
+
+```
+Basic <BASE64_ENCODED_APPLICATION_SECRET>
+```
+
+where `BASE64_ENCODED_APPLICATION_SECRET` is the base 64 encoding of the string `client_id:client_secret`. The *client_id* and *client_secret*.  
+
+For example, by using `client_id` *application* and `client_secret` *secret*, we would have the authorization header `Basic YXBwbGljYXRpb246c2VjcmV0`.
+
+## 1. Authenticate by password
+
+Send a `POST` with the following body:
```

Comment:
I think we could add a CURL request as example to have more meaning when you are saying _Send a `POST`_

What do you think?

Reply:
added